docs/source/export.rst

```diff
@@ -1,11 +1,293 @@
+.. _torch.export:
+
 torch.export
 =====================
 
-.. TODO: Add torch.export() tutorial here.
-
 .. warning::
     This feature is a prototype and may have compatibility breaking changes in the future.
 
+
+Overview
+--------
+
+``torch.export`` is an ahead-of-time (AOT) process built to fully convert
+define-by-run PyTorch code into a static and standardized model representation,
+with strong soundness guarantees. It is designed to streamline and standardize the
+way PyTorch interacts with various systems and platforms.
```

Comment:
The words are kind of big - "why is export a 'process'"? "what does define-by-run mean?" (I'm still not sure after all these years).

A more down to earth explanation could be something like "torch.export is a collection of APIs for sound full program capture of a PyTorch model into a standardized representation"